lib/miq_automation_engine/service_models/miq_ae_service_vm_or_template.rb

```diff
@@ -41,7 +41,7 @@ class MiqAeServiceVmOrTemplate < MiqAeServiceModelBase
     expose :compliances,           :association => true
     expose :last_compliance,       :association => true
     expose :ems_events,            :association => true
-
+    expose :evacuate
```

Comment:
I believe, that there should be ```expose :evacuate, :override_return => nil```

A return value of the method https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/openstack/cloud_manager/vm/operations/relocation.rb#L37 does not look useful for Automate.